dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -725,6 +725,11 @@ private void run(String[] argv) {
                         argsArray = args.toArray(argsArray);
                         handled = DepCommand.handleDepCommand(argsArray);
                     } else if (YAML.equals(mode)) {
+                        if (!DEBUG.get() && !INFO.get()) {
+                            // Switching log off if DEBUG or INFO is not set
+                            // yamlClient will generate unnecessary error logs
+                            root.setLevel(Level.OFF);
```

Comment:
Kinda wonder if you can set the log level for a particular package or lower in the tree. Probably not too important since this only affects this specific command

Reply:
I didn't know this until looking at this PR, but there's already code above starting around line 677 that invokes `root.setLevel()` based on the command line switches, and it feels weird to me to set it to `Level.OFF` only for this particular path.

I'd suggest one of these:

1. Doing `root.setLevel(Level.OFF)` up above instead.
2. Don't use LOG.error for errors, use LOG.info
3. Do a pattern something like this:

```
if (LOG.isDebugEnabled()) { 
   LOG.error("the error message", theException);
}
```

I dislike 2.

I lean towards 3 -- it seems a little counter-intuitive, but the nice thing about it is that it won't log by default, but if it does log, it's logging at the ERROR level, so it will stand out for developers. 1 seems a little severe, it's also possible.

Also, see dockstore/dockstore#5223, which relates to this one -- ideally the solution would work for both.